ledger-core/v2/Makefile.testing

```diff
@@ -14,13 +14,12 @@ test_unit: ## run all unit tests
 
 .PHONY: test_func
 test_func: ## run functest FUNCTEST_COUNT times
-	CGO_ENABLED=1 $(GOTEST) -test.v $(TEST_ARGS) -tags "functest bloattest" ./application/functest -count=$(FUNCTEST_COUNT)
```

Comment:
Can't find tests with this tag.

Reply:
bloattests were tests that bloat ledger with everlasting abandoned requests, no ledger for now, no bloattests :)